airflow/example_dags/example_kubernetes_executor_config.py

```diff
@@ -87,8 +87,8 @@ def test_volume_mount():
         python_callable=print_stuff,
         executor_config={
             "KubernetesExecutor": {
-                "securityContext": {
-                    "runAsUser": 1000
+                "labels": {
+                    "runAsUser": "1000"
```

Comment:
Are you saying per your other comment  that this does not set runAsUser to 1000?

(For this please pick a different label that doesn't look like it might do something :slightly_smiling_face: )

Reply:
Exactly, if you're in support of removing `securityContext` I'll change [the comment](https://github.com/apache/airflow/blob/0499bab86e5045fd18683e64ae471d06ad6c806f/airflow/example_dags/example_kubernetes_executor_config.py#L84) as well.

Reply:
Not really in favour of removing the feature -- that seems like a massive bug that we need to fix.

And if it doesn't work there's no point having that example task.

Reply:
Ok, so continue supporting passing the argument even though it doesn't set the user? I don't know if it's a bug exactly. You can pass `security_context` and that will work to set user.